index.js

```diff
@@ -30,6 +30,8 @@ const releaseTitle = "im-" + finalVersionNumber;
 const jiraUrlStrip = jiraBaseUrl.match(/https:\/\/(\S*)/);
 const jiraUrl = jiraUrlStrip[1];
 const pattern = /[a-zA-Z]{2,}-[0-9]+/gm;
+const trimPreviousVersionNumber = previousVersionNumber.trim();
+const trimReleaseBranch = releaseBranch.trim();
```

Comment:
Same here, will crash

Reply:
Thank you for the review Simon, I have moved them after the checks for the env variables being set.